content/_includes/paypal-button.html

```diff
@@ -0,0 +1,5 @@
+<form action="https://www.paypal.com/cgi-bin/webscr" method="post" target="_top" style="display: inline;">
+  <input type="hidden" name="cmd" value="_s-xclick">
+  <input type="hidden" name="hosted_button_id" value="Z55VS5LBBSZTJ">
+  <input type="image" class="paypal-btn" src="/assets/img/donate-button2.png" name="submit" alt="Donate to Common Workflow Language via PayPal">
+</form>
```

Comment:
Maybe we could leave the donate and social buttons in some grey tone, or monochrome. Right now I think these three elements call more my attention than the logo on the left-hand side (and not immediately visible anyway when on mobile/tablet mode I think).

Reply:
Hmm,  I see your point. And yes, they're at the bottom of the nav menu on tablet/mobile.

@swzCuroverse @mr-c any thoughts on whether or not this makes sense to change?

Reply:
> Maybe we could leave the donate and social buttons in some grey tone, or monochrome. Right now I think these three elements call more my attention than the logo on the left-hand side 

monochrome or gray is fine with me, thanks!

Reply:
Fwiw, I think it's actually the Donate button that's drawing the attention, rather than the icons. Here's a few rough example options:

1. Donate text link w/ color social icons (my preference):

![donate-link-with-color-icons-1330x864](https://user-images.githubusercontent.com/10420402/144466691-47cc64f0-71ff-4a75-b8a0-c199cd23c84f.png)
<br>

2. Donate button w/ grey tone social icons (still visually overpowering):

![donate-button-with-grey-tone-icons-1330-864](https://user-images.githubusercontent.com/10420402/144466726-a2c4dd5f-63bf-4bc9-8bee-9278049c0536.png)
<br>

3. Donate text link w/ grey tone social icons:

![donate-link-with-grey-tone-icons-1330x864](https://user-images.githubusercontent.com/10420402/144466759-fbaa1825-3c36-4a3a-893d-6613ed8551d8.png)


@swzCuroverse @mr-c any preference?

Reply:
I lean towards option 3 ; thanks!

Reply:
New icon/colors look great! I don't feel like it's distracting now.

@lunacodes are the menu items right-aligned? On my 15" laptop is seems to have a bit of extra space. I thought the menu would be either left aligned, with some icons like donate/gh/twitter being right aligned, or centralized?

![image](https://user-images.githubusercontent.com/304786/144985674-e7e19572-e2b4-4353-8632-9cae08279eed.png)

Reply:
@kinow Yep - the menu items are (and have always been) right-aligned. Right now, it wouldn't work well left-aligned, due to the number of menu item. Here's a screenshot of what it would look like left-aligned:

![cwl-menu-left-aligned](https://user-images.githubusercontent.com/10420402/145316510-f855fac4-669e-48da-9821-3cf112689633.png)

With a left align, I'd probably move the logo over so it lines up with the page title. However, if I do that here, with the number of menu items, it's too cluttered, imo. Two rough screenshots:

![cwl-menu-left-aligned3](https://user-images.githubusercontent.com/10420402/145317102-1d358800-6dba-4131-826d-3b3f2b045f75.png)

![cwl-menu-left-aligned2](https://user-images.githubusercontent.com/10420402/145317098-6ce4d338-ed9b-4375-9607-67aeab6a4525.png)

I don't really see any benefit to moving it rn, unless the number of menu items decreases - in which case, it would work ok either way.

Reply:
>I don't really see any benefit to moving it rn, unless the number of menu items decreases - in which case, it would work ok either way.

:+1: